scripts/gen.py

```diff
@@ -597,6 +599,14 @@ def get_return_type_str(t, orig_sig):
   return orig_sig[0:token.column - 2]
 
 
+def generate_debug_code(t, fname, rname, params, param_vars, ref_param):
```

Comment:
This isn't really debug, it's metrics. Maybe `generate_metrics_code`?

Reply:
It is supposed to generate VLOG debug as well. IMO the name is OK. After all even the metrics are a debugging tool.